README.md

```diff
@@ -1,43 +1,43 @@
-## CloudFoundry PHP Example Application: Drupal 
+## cloud.gov example application: Drupal 7
 
-This is an example application which can be run on CloudFoundry using the [PHP Build Pack].
+This is an example application that can be run on [cloud.gov](https://www.cloud.gov) using the [PHP Buildpack](http://docs.cloudfoundry.org/buildpacks/php/index.html).
 
-This is an out-of-the-box implementation of Drupal.  It's an example of how common PHP applications can easily be run on CloudFoundry.
+This is an out-of-the-box implementation of Drupal 7.  It's an example of how common PHP applications can easily be run on cloud.gov.
 
 ### Usage
 
 1. Clone the app (i.e. this repo).
 
-  ```bash
-  git clone https://github.com/cloudfoundry-samples/cf-ex-drupal.git cf-ex-drupal
-  cd cf-ex-drupal
-  ```
+    ```bash
+    git clone https://github.com/18F/cf-ex-drupal.git cf-ex-drupal
+    cd cf-ex-drupal
+    ```
 
-1.  If you don't have one already, create a MySQL service.  With Pivotal Web Services, the following command will create a free MySQL database through [ClearDb].  Any MySQL provider should work.
+1.  If you don't have one already, create a MySQL service. If you have a cloud.gov account, the following command will create a free MySQL database.
 
-  ```bash
-  cf create-service cleardb spark mysql
-  ```
+    ```bash
+    cf create-service aws-rds shared-mysql my-db-service
+    ```
 
-1. Edit `sites/default/settings.php` and change the `drupal_hash_salt`.  This should be uniqe for every installation.  Optionally edit any other settings, however you do *not* need to edit the database configuration.  The file included with this example will automatically pull that information from `VCAP_SERVICES`.
+1. Edit `htdocs/sites/default/settings.php` and change the `drupal_hash_salt`.  This should be unique for every installation.  Optionally edit any other settings, however you do *not* need to edit the database configuration.  The file included with this example will automatically pull that information from `VCAP_SERVICES`.
 
-1. Push it to CloudFoundry.
+1. Push it to cloud.gov.
 
-  ```bash
-  cf push
-  ```
```

Comment:
Ha, the `--random-route` option is pretty fun if you haven't run into it before - when you set that flag, it automatically adds a couple random words to the end of the hostname, like `cf-ex-drupal-fancy-sloth`, so that you don't have to worry about it.